source/_components/light.yeelight.markdown

```diff
@@ -196,11 +197,11 @@ light:
 
 ### {% linkable_title Custom effects %}
 
-This example shows how you can add your custom effects in your configuration.
+This example shows how you can add your custom effects in your configuration. To turn on you can use [light.turn_on](/components/light/#service-lightturn_on) service.
```

Comment:
To turn on the light .. ? The sentence sounds a bit off.

Reply:
Fixed